common/app/common/NewNavigation.scala

```diff
@@ -216,34 +216,50 @@ object NewNavigation {
     val name = ""
 
     val uk = NavLinkLists(List(
-      apps.copy(url = apps.url + "?INTCMP=apps_uk_web_newheader"),
       jobs.copy(url = jobs.url + "?INTCMP=jobs_uk_web_newheader"),
       dating.copy(url = dating.url + "?INTCMP=soulmates_uk_web_newheader"),
-      NavLink("professional", "/guardian-professional"),
-      masterClasses.copy(url = masterClasses.url + "?INTCMP=masterclasses_uk_web_newheader"),
-      NavLink("courses", "/?INTCMP=NGW_TOPNAV_UK_GU_COURSES"),
       NavLink("holidays", "https://holidays.theguardian.com/?utm_source=theguardian&utm_medium=guardian-links&utm_campaign=topnav&INTCMP=topnav"),
-      todaysPaper, observer, crosswords
+      NavLink("courses", "https://courses.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_COURSES"),
+      ukMasterClasses.copy(url = ukMasterClasses.url + "?INTCMP=masterclasses_uk_web_newheader"),
+      NavLink("professional networks", "/guardian-professional"),
+      apps.copy(url = apps.url),
+      podcasts,
+      video,
+      newsletters,
+      todaysPaper,
+      observer,
+      crosswords
     ))
 
     val au = NavLinkLists(List(
-      apps.copy(url = apps.url + "?INTCMP=apps_au_web_newheader"),
-      masterClasses.copy(url = masterClasses.url + "?INTCMP=masterclasses_au_web_newheader"),
-      crosswords, video
+      jobs.copy(url = jobs.url + "/landingpage/2868291/jobs-australia-html/?INTCMP=jobs_au_web_newheader"),
+      auMasterClasses.copy(url = auMasterClasses.url + "?INTCMP=masterclasses_au_web_newheader"),
```

Comment:
If the au master class url is unique and not used in any other editions, then you don't have to copy it. You can just add that campaign code in the url when you defined it. For example

```
 val auMasterClasses = NavLink("masterclasses", "/guardian-masterclasses-australia?INTCMP=masterclasses_au_web_newheader")
```